Framework/Algorithms/src/MaxEnt/MaxentCalculator.cpp

```diff
@@ -271,7 +284,7 @@ void MaxentCalculator::iterate(const std::vector<double> &data,
           m_coeffs.c2[k][l] += directionsDat[k][i] * directionsDat[l][i] /
                                m_errors[i] / m_errors[i];
       }
-      m_coeffs.c2[k][l] *= 2.0 / chiSq;
+      m_coeffs.c2[k][l] *= 2.0 / fac;
```

Comment:
this would be a factor of 4 larger. What is the logic?

Reply:
To make the quadratic function be equal to the actual chi squared in the search subspace. Without that they were different.